src/runner.js

```diff
@@ -4,13 +4,24 @@ const parser = require('./parser/parser.js');
 const results = require('./results.js');
 const DefaultVariableStorage = require('./default-variable-storage.js');
 const nodeTypes = require('./parser/nodes.js').types;
+const MessageFormat = require('messageformat');
 
 class Runner {
-  constructor() {
+  /**
+   * Creates an instance of Runner.
+   * @param {object} [options] Options for the runner
+   * @param {object} [options.formater] Changing default formater used (MessageFormat)
+   * @param {string} [options.language] Options for the language of the formater
+   * @memberof Runner
+   */
+  constructor(options) {
     this.yarnNodes = {};
     this.variables = new DefaultVariableStorage();
     this.functions = {};
-    this.visited = {}; // Which nodes have been visited
+    this.visited = {}; // Which nodes have been visitedthis.options = options;
+    this.options = options;
```

Comment:
I suppose here we should say
`this.options = options || {};`

since options being undefined would make it impossible to attach defaults to

Reply:
yeah